packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -404,7 +406,7 @@ export const Table = <
       newState: TableState<T>,
       action: ActionType,
       previousState: TableState<T>,
-      instance?: TableInstance<T>,
+      instance: TableInstance<T>,
```

Comment:
Earlier, instance was optional. But since `onShiftSelectHandler` on line 434 needed a non-null instance variable, I made it required. Is this alright?

Reply:
I don't think this is the right way to do it. Can you find out whether it is actually optional? Might need to look in the documentation/codebase of react-table.

Because if it is not always passed in the JavaScript code and we remove the `?`, then we are lying about it.